worlds/bomb_rush_cyberfunk/__init__.py

```diff
@@ -0,0 +1,203 @@
+    def __init__(self, multiworld: MultiWorld, player: int):
+        super(BombRushCyberfunkWorld, self).__init__(multiworld, player)
+        self.item_classification: Dict[BRCType, ItemClassification] = {
+            BRCType.Music: ItemClassification.filler,
+            BRCType.GraffitiM: ItemClassification.progression,
+            BRCType.GraffitiL: ItemClassification.progression,
+            BRCType.GraffitiXL: ItemClassification.progression,
+            BRCType.Outfit: ItemClassification.filler,
+            BRCType.Character: ItemClassification.progression,
+            BRCType.REP: ItemClassification.progression_skip_balancing,
+            BRCType.Camera: ItemClassification.progression
+        }
+
+
+    def set_rules(self):
+        rules(self)
+
+
+    def get_item_classification(self, item_type: BRCType) -> ItemClassification:
+        classification = ItemClassification.filler
+        if item_type in self.item_classification.keys():
+            classification = self.item_classification[item_type]
+
+        return classification
+
+
+    def create_item(self, name: str) -> "BombRushCyberfunkItem":
+        item_id: int = self.item_name_to_id[name]
+        item_type: BRCType = self.item_name_to_type[name]
+        classification = self.get_item_classification(item_type)
+
+        return BombRushCyberfunkItem(name, classification, item_id, self.player)
+
+
+    def create_event(self, event: str) -> "BombRushCyberfunkItem":
+        return BombRushCyberfunkItem(event, ItemClassification.progression_skip_balancing, None, self.player)
+    
+
+    def get_filler_item_name(self) -> str:
+        item = self.random.choice(item_table)
+
+        while self.get_item_classification(item["type"]) == ItemClassification.progression:
+            item = self.random.choice(item_table)
+
+        return item["name"]
+
+
+    def generate_early(self):
+        if self.options.starting_movestyle == StartStyle.option_skateboard:
+            self.item_classification[BRCType.Skateboard] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.Skateboard] = ItemClassification.progression
+
+        if self.options.starting_movestyle == StartStyle.option_inline_skates:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.progression
+        
+        if self.options.starting_movestyle == StartStyle.option_bmx:
+            self.item_classification[BRCType.BMX] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.BMX] = ItemClassification.progression
+
+
+    def create_items(self):
+        rep_locations: int = 87
+        if self.options.skip_polo_photos:
+            rep_locations -= 18
+
+        self.options.total_rep.round_to_nearest_step()
+        rep_counts = self.options.total_rep.get_rep_item_counts(self.multiworld.random, rep_locations)
```

Comment:
```suggestion
        rep_counts = self.options.total_rep.get_rep_item_counts(self.random, rep_locations)
```
Recommended to use self.random instead of self.multiworld.random so that things stay deterministic